Notebooks/Spark.NET C#/GitHub Projects Advanced.ipynb

```diff
@@ -0,0 +1,576 @@
+{
+  "metadata": {
+    "description": "More advanced example of analyzing and extracting insights from GitHub data.",
+    "saveOutput": true,
+    "kernelspec": {
+      "name": "sparkdotnetkernel",
+      "display_name": "Spark .NET"
+    },
+    "language_info": {
+      "name": "csharp"
+    }
+  },
+  "nbformat": 4,
+  "nbformat_minor": 2,
+  "cells": [
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "outputs": [],
+      "metadata": {},
+      "source": [
+        "spark"
+      ],
+      "attachments": {}
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 2,
+      "outputs": [
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/html": "<div><br /><font color=\"purple\" size=\"18\">GitHub &#128150; .NET for Apache Spark</font><hr></hr><div><font size=\"4\">Welcome to the Spark.NET Notebook E2E Playground!</font><br /><br /><font size=\"4\">Let&#39;s dive into the world of analyzing GitHub meta-data using .NET for Apache Spark.</font><br /><br /><font size=\"4\">You can see here that we&#39;re able to use</font><font color=\"purple\" size=\"4\"> HTML </font><font size=\"4\">to render beautiful cells.</font></div></div>"
+          },
+          "execution_count": 2,
+          "metadata": {}
+        }
+      ],
+      "metadata": {
+        "cell_status": {
+          "execute_time": {
+            "duration": 1280.983154296875,
+            "end_time": 1571951678785.414
+          }
+        },
+        "collapsed": false
+      },
+      "source": [
+        "display(\n",
+        "    div(\n",
+        "        br(),\n",
+        "        font[size: 18, color: \"purple\"](\"GitHub 💖 .NET for Apache Spark\"),\n",
+        "        hr(),\n",
+        "        div(\n",
+        "            font[size: 4](\"Welcome to the Spark.NET Notebook E2E Playground!\"),\n",
+        "            br(), br(),\n",
+        "            font[size:4](\"Let's dive into the world of analyzing GitHub meta-data using .NET for Apache Spark.\"),\n",
+        "            br(), br(),\n",
+        "            font[size:4](\"You can see here that we're able to use\"),font[size:4, color:\"purple\"](\" HTML \"),\n",
+        "            font[size:4](\"to render beautiful cells.\")\n",
+        "        )\n",
+        "    ));"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## <font color=purple>Read Input Data 📈 </font>\n",
+        "We'll read in data about various GitHub activity to understand trends and gain some interesting insights.\n",
+        "### We can start off reading a csv file containing commit information. Be sure to update the following cells with the correct data file paths."
```

Comment:
@AFFogarty Here is where I included a message indicating for users to update their file paths. If you have any ideas to improve, please let me know.